### DISCUSSION TITLE
升级到2.0.4-fix2版无法读取外网的mysql，望大神修正

### DISCUSSION THREAD

**Discussion by jmheqiao** [AlistGo/alist#280] [General]
我原先在2.0.3版配置的网上免费mysql，在升级到2.0.4-fix2后无法读取，只好用回本地配置。同一mysql配置，我再次安装2.0.3和2.0.4-fix后又可以读取mysql，只有2.0.4-fix2不正常，。我是用armbian部署的，mysql是网上的免费申请的。请大神修正！

**Comment by xhofe** [comment]
请提供错误日志

**Comment by xhofe** [comment]
应该同 #281

**Comment by jmheqiao** [comment]
对，错误是一样的
Q[36mINFO[0m[2022-01-04 23:18:53] init log...                                  
Q[36mINFO[0m[2022-01-04 23:18:53] register driver: [123Pan]                    
Q[36mINFO[0m[2022-01-04 23:18:53] register driver: [189Cloud]                  
Q[36mINFO[0m[2022-01-04 23:18:53] register driver: [AliDrive]                  
Q[36mINFO[0m[2022-01-04 23:18:53] register driver: [Alist]                     
Q[36mINFO[0m[2022-01-04 23:18:53] register driver: [FTP]                       
Q[36mINFO[0m[2022-01-04 23:18:53] register driver: [GoogleDrive]               
Q[36mINFO[0m[2022-01-04 23:18:53] register driver: [Lanzou]                    
Q[36mINFO[0m[2022-01-04 23:18:53] register driver: [Native]                    
Q[36mINFO[0m[2022-01-04 23:18:53] register driver: [Onedrive]                  
Q[36mINFO[0m[2022-01-04 23:18:53] register driver: [PikPak]                    
Q[36mINFO[0m[2022-01-04 23:18:53] register driver: [S3]                        
Q[36mINFO[0m[2022-01-04 23:18:53] register driver: [ShandianPan]               
Q[36mINFO[0m[2022-01-04 23:18:53] register driver: [WebDav]                    
S[36mINFO[0m[2022-01-04 23:18:53] reading config file: /alist/config/config.json 
Q[36mINFO[0m[2022-01-04 23:18:53] init cron...                                 
Q[36mINFO[0m[2022-01-04 23:18:53] init model...                                
Q[36mINFO[0m[2022-01-04 23:18:53] auto migrate model...                        
Q[36mINFO[0m[2022-01-04 23:18:54] init settings...                             
�[31mFATA[0m[2022-01-04 23:19:00] failed write setting: Error 1366: Incorrect string value: '\xF0\x9F\x8F\xA0' for column 'value' at row 1
